config/routes.rb

```diff
@@ -686,4 +686,10 @@ def draw(routes_name)
     get 'wopi/files/:id', to: 'wopi#file_get_endpoint', as: 'wopi_rest_endpoint'
     post 'wopi/files/:id', to: 'wopi#post_file_endpoint'
   end
+
+  if Rails.env.test?
+    namespace :wopi do
+      post :post_file_endpoint
```

Comment:
What's the reason behind this? Wouldn't

```
post :post_file_endpoint, subdomain: ENV['WOPI_SUBDOMAIN']
```
work in the tests? or just setting `ENV['WOPI_SUBDOMAIN']` to nil?

Reply:
Fix